backend/Cargo.toml

```diff
@@ -43,3 +43,7 @@ utoipa-scalar = { version = "0.3.0", features = ["actix-web"] }
 # Custom crates
 migration = { path = "migration" }
 regex = "1.11.1"
+
+[dev-dependencies]
+testcontainers = "0.23.1"
```

Comment:
Mention it in `Backend/README.md`

Reply:
Inform about security, because most of people will be concerned why it asks for sudo to tests and mention better alternatives like podman. Also add note about the custom use flag for those sests